code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java

```diff
@@ -207,9 +206,6 @@ protected void allocate(Instance instance) {
 
     protected void storage(Instance instance, ProcessState state) {
         List<Volume> volumes = getObjectManager().children(instance, Volume.class);
-
-        volumes.addAll(InstanceHelpers.extractVolumesFromMounts(instance, objectManager));
```

Comment:
Yeah...after seeing it and thinking about what it did, I came to the conclusion that it should not be there.